src/java/org/apache/cassandra/utils/TimeUUID.java

```diff
@@ -411,9 +437,28 @@ public static byte[] nextTimeUUIDAsBytes()
             return toBytes(rawTimestampToMsb(unixMicrosToRawTimestamp(nextUnixMicros())), clockSeqAndNode);
         }
 
+        public static byte[] nextTimeUUIDAsBytes(long unixMicrosAtLeast)
```

Comment:
this is 100% unsafe in accord as it is:

1) based off the local node, doesn't act the same cross the cluster
2) shared by non-accord tables, so both are corrupting each other...

You really don't want this and instead want `unsafe(unixMicros)` which does not use any state...

Now, you still have the issue that the `clockSeqAndNode` isn't the same, so you have to solve that as well... if we used the instance that created the txn, then that would make sense... else we would need to have a global one used by all accord nodes so its consistent...

@belliottsmith thoughts?